test/e2e/e2eutil/wait_util.go

```diff
@@ -0,0 +1,144 @@
+package e2eutil
```

Comment:
Maybe rename this file to "util.go"? We can put all methods that's not very specific into here.